### DISCUSSION TITLE
可以搞个bilibili的规则吗

### DISCUSSION THREAD

**Discussion by sgr997** [xiaohucode/yidaRule#43] [Ideas]
软件挺好用，感谢作者大大，bilibili上有很多听书，歌曲资源哈哈，可惜截图里这个软件不支持ios平台
![image](https://github.com/xiaohucode/yidaRule/assets/51942582/5c362703-ddcb-400e-88ac-d965e6e112e7)

**Comment by xiaohucode** [comment]
不太行,b站的视频音频是分离的，目前只有音频能正常播放,视频没有声音

**Reply by sgr997** [reply]
就是只播放音频就行，就可以听书和听音乐了

**Reply by xiaohucode** [reply]
> 就是只播放音频就行，就可以听书和听音乐了

最新的规则已包含bilibili 规则，重新导入即可

**Comment by sgr997** [comment]
> 不太行,b站的视频音频是分离的，目前只有音频能正常播放,视频没有声音

nice，感谢大佬，好用的很![image](https://github.com/xiaohucode/yidaRule/assets/51942582/556e9ee4-e178-4a4b-8af2-c9e260d09da0)
